packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -445,101 +445,102 @@ Describes an IP address used in a request.
-| application.user_membership.add                | A user was assigned to an application.                     |
-| application.user_membership.change_username    | The name of a user assigned to an application was changed. |
-| application.user_membership.remove             | A user was removed from an application.                    |
+| application.lifecycle.activate                 | An application is activated.                              |
+| application.lifecycle.create                   | An application is created.                                |
+| application.lifecycle.deactivate               | An application is deactivated.                            |
+| application.lifecycle.delete                   | An application is deleted.                                |
+| application.lifecycle.update                   | An application is updated.                                |
+| application.user_membership.add                | A user is assigned to an application.                     |
+| application.user_membership.change_username    | The name of a user who is assigned to an application is changed. |
+| application.user_membership.remove             | A user is removed from an application.                    |
 
-### Group Event
+### Group event
 
 | Event                          | Description                      |
 | :----------------------------- | :------------------------------- |
-| group.user_membership.add      | A user was added to a group.     |
-| group.user_membership.remove   | A user was removed from a group. |
+| group.user_membership.add      | A user is added to a group.     |
+| group.user_membership.remove   | A user is removed from a group. |
 
-### Policy Events
+### Policy events
 
 | Event                         | Description                         |
 | :---------------------------- | :---------------------------------- |
-| policy.lifecycle.activate     | A rule in a policy was activated.   |
-| policy.lifecycle.create       | A rule in a policy was created.     |
-| policy.lifecycle.deactivate   | A rule in a policy was deactivated. |
-| policy.lifecycle.delete       | A rule in a policy was deleted.     |
-| policy.lifecycle.update       | A rule in a policy was updated.     |
-| policy.rule.activate          | A rule in a policy was activated.   |
-| policy.rule.add               | A rule was added to a policy.       |
-| policy.rule.deactivate        | A rule in a policy was deactivated. |
-| policy.rule.delete            | A rule was deleted from a policy.   |
-| policy.rule.update            | A rule in a policy was updated.     |
-
-#### Policy Event Details
-
-* `policy.evaluate_sign_on` provides context on the values used and evaluated in the context of the Okta sign on policy. For example, you can determine which network zones were matched for this event.
+| policy.lifecycle.activate     | A rule in a policy is activated.   |
```

Comment:
I would check on the descriptions here ...  there are the same descriptions for different events. E.g., `policy.lifecycle.acitvate` had a description of "A rule in a policy is activated.", which is the same description for `policy.rul.activate`. (The description sounds applicable for the latter event.)

For the first five "non-rule" events, I'm guessing we should remove "rule" as part of the description. That is, `policy.lifecycle.activate` is probably "A lifecycle policy is activated".